packages/react-rest/README.md

```diff
@@ -700,11 +815,15 @@ export default App
 > **Note**
 > This is the extent of functionality the mock service worker supports. If you would like to continue exploring the HatchifyJS ecosystem, then we suggest setting up a standalone HatchifyJS backend. The getting started guide for a HatchifyJS backend can be found [here](https://github.com/bitovi/hatchify/blob/main/packages/koa/README.md).
```

Comment:
>This is the extent...

I'm not sure what "This" is referring to here. I think we need to be a bit more explicit about which limits of msw we've reached. If someone could please provide me with some details here, I'll go ahead and add them

Reply:
our msw implementation for relationships (and in general) is pretty barebone - you can create w/ relationships (todo w/ user) but you can't create a user, you can't edit todos/users, you can't remove relationship without deleting the record etc. so however you can word that so it makes sense go for it 😄

Reply:
Updated!